web-packages/imgui-demo/src/panels/node-graph/tools-graph/nodes/state-tools-node.ts

```
@@ -0,0 +1,419 @@
+    const scaledRounding = 3.0 * ctx.m_viewScaleFactor;
+
+    const DrawStateTypeWindow = (ctx: DrawContext, iconColor: Color, fontColor: Color, width: number, pIcon: string, pLabel: string) => {
+      this.BeginDrawInternalRegion(ctx);
+
+      {
+        // ImGuiX.ScopedFont(ImGuiX.Font.Small, iconColor);
+        // ImGui.Text(pIcon);
+      }
+
+      {
+        // ImGuiX.ScopedFont(ImGuiX.Font.Small, fontColor);
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)